SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -3,24 +3,23 @@
 #include "CoreMinimal.h"
 
 #include "Utils/SchemaOption.h"
+#include "SpatialCommonTypes.h"
 
 #include <cstdint>
 
-using Worker_EntityId = std::int64_t;
-
 class USpatialPackageMapClient;
 
 struct SPATIALGDK_API FUnrealObjectRef
 {
 	FUnrealObjectRef() = default;
 	FUnrealObjectRef(const FUnrealObjectRef&) = default;
 
-	FUnrealObjectRef(Worker_EntityId Entity, uint32 Offset)
+	FUnrealObjectRef(Worker_EntityId_Key Entity, uint32 Offset)
```

Comment:
Why has this changed? Entity ID widths haven't changed and this doesn't seem like it;s using a map.

Reply:
It gave me an error once, but I reran the build with these Key changes reverted, and it works now. The original error was probably a false alarm caused by a change elsewhere, I will revert these now.